CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ This version of the dashboard requires Cumulus API >= v14.0.0
   - Added action to list pages to refresh cumulus db connection
   - Updated `git+ssh://git@` to `git+https://` for npm 8.6.0 generated package-lock.json,
     and this is an [known issue](https://github.com/npm/cli/issues/2610) with npm 8.6.0
+- **CUMULUS-3107**
+  - Added d3-dag dependency to replace dagre-d3
```

Comment:
degre-d3 isn't replaced in this PR?

Reply:
Please move CUMULUS-3107 to unreleased section.

Reply:
The d3-dag dependency was added to replace dagre-d3 in the ticket https://bugs.earthdata.nasa.gov/browse/CUMULUS-3184

Reply:
Ok if this work is not done, we should remove this entry.

Reply:
The Unreleased section is on the top of the changelog
```
## [Unreleased]

### Changed
- **CUMULUS-3107**
```